src/main/resources/db/migration/all/20220803121542__revise_service_column_default_value.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE premises ALTER COLUMN service SET DEFAULT 'approved-premises';
```

Comment:
This won't work as intended because the rows already exist and already have a value of 'CAS1' - so needs to be something like:
```
UPDATE premises SET service = 'approved-premises' WHERE service='CAS1';
UPDATE premises SET service='temporary-accommodation' WHERE service='CAS3';
```

Reply:
Doh of course